litestar/security/jwt/auth.py

```diff
@@ -472,35 +427,6 @@ def openapi_components(self) -> Components:
             }
         )
 
-    @property
-    def middleware(self) -> DefineMiddleware:
```

Comment:
backwards compatibility; see earlier comment

Reply:
done, on the base